core/interfaces.go

```diff
@@ -82,6 +85,9 @@ type ValidatorAPI interface {
 	// RegisterPubKeyByAttestation registers a function to query validator by attestation.
 	RegisterPubKeyByAttestation(func(ctx context.Context, slot, commIdx, valCommIdx int64) (PubKey, error))
 
+	// RegisterGetDutyFunc registers a function to query duty data.
+	RegisterGetDutyFunc(func(ctx context.Context, duty Duty) (FetchArgSet, error))
```

Comment:
same here, remove param names. `RegisterPubKeyByAttestation` has names, since there are `3x int64` which is not clear.

Reply:
done